include/swift/Basic/PrimarySpecificPaths.h

```diff
@@ -0,0 +1,43 @@
+//===--- PrimarySpecificPaths -----------------------------------*- C++ -*-===//
```

Comment:
Nitpick: missing .h.

(Once upon a time these headers were for printouts. I know, right?)

Reply:
Fixed.